js/form.js

```diff
@@ -0,0 +1,193 @@
+  effectImage.style.filter = 'blur(3px)';
+  effectLevelPin.style.left = 455 + 'px';
+  effectLevelVal.style.width = 455 + 'px';
+  effect = 'blur';
+  getEffectLevel();
+  effectLevel.classList.remove('hidden');
+});
+
+effectHeat.addEventListener('click', function() {
+  effectImage.style.filter = 'brightness(3)';
+  effectLevelPin.style.left = 455 + 'px';
+  effectLevelVal.style.width = 455 + 'px';
+  effect = 'brightness';
+  getEffectLevel();
+  effectLevel.classList.remove('hidden');
+});
+
+hashtagsForm.addEventListener('change', function() {
+  var valHash = hashtagsForm.value;
+  var arrHash = valHash.split(' ');
+  var errorHash = false;
+
+  if (arrHash.length > 5){
+    hashtagsForm.setCustomValidity('не больше пяти хэштегов');
+    errorHash = true;
+  }
+  console.log(arrHash.length);
+
+  for (var hash of arrHash) {
+    if (hash[0] !== "#") {
+      hashtagsForm.setCustomValidity('хештег должен начинаться с #');
+      errorHash = true;
+    }
+    if (hash.length < 2) {
+      hashtagsForm.setCustomValidity('Хэштег должен состоять минимум из 2х символов');
+      errorHash = true;
+    } else if (hash.length > 20) {
+      hashtagsForm.setCustomValidity('Хэштег не должен превышать 20 символов');
+      errorHash = true;
+    }
+  }
+
+  for (var i = 0; i < arrHash.length; i++) {
+    var arrSort = arrHash.sort();
+    if (arrSort[i] === arrSort[i + 1]) {
+      hashtagsForm.setCustomValidity('Одинаковые хэштеги');
+      errorHash = true;
+    }
+  }
+
+  if (errorHash === false) {
+    hashtagsForm.setCustomValidity('');
+  }
+});
+
+hashtagsForm.style.textTransform = 'lowerCase';
+
+var newLeft;
+var level;
+var effect;
+
+effectLevelPin.onmousedown = function(evt) {
+  evt.preventDefault();
+
+  var shiftX = evt.clientX - effectLevelPin.getBoundingClientRect().left;
+
+  document.addEventListener('mousemove', onMouseMove);
+  document.addEventListener('mouseup', onMouseUp);
+
+  function onMouseMove(evt) {
+    newLeft = evt.clientX - shiftX - effectLevelLine.getBoundingClientRect().left;
+
+    if (newLeft < 0) {
+      newLeft = 0;
+    }
+    var rightEdge = effectLevelLine.offsetWidth;
+    if (newLeft > rightEdge) {
+      newLeft = rightEdge;
+    }
+
+    effectLevelPin.style.left = newLeft + 'px';
+    effectLevelVal.style.width = newLeft + 'px';
+    getEffectLevel();
+
+  }
+
+  function onMouseUp() {
+    document.removeEventListener('mouseup', onMouseUp);
+    document.removeEventListener('mousemove', onMouseMove);
+  }
+
+};
+
+  function getEffectLevel() {
+    level = newLeft/455;
+    if (effect === 'grayscale') {
+    effectImage.style.filter = effect + '('+ level +')';
+    } else if (effect === 'sepia') {
+      effectImage.style.filter = effect + '('+ level +')';
+    } else if (effect === 'invert') {
+      effectImage.style.filter = effect + '('+ (level * 100) + '%' +')';
+    } else if (effect === 'blur') {
+      effectImage.style.filter = effect + '('+ (level + 0.1) * 3 + 'px' + ')';
```

Comment:
```suggestion
      effectImage.style.filter = effect + '('+ ((level * 2) + 1)  + 'px' + ')';
```
если значение должно вычисляться в пределах от 1 до 3, то правильно будет вычислять его вот так